docs/faq.rst

```diff
@@ -209,30 +232,24 @@ If you want to place your package in a different directory or have multiple
 python packages in the same project, then create a pip requirements file. You
```

Comment:
In the previous paragraph, we should probably change

```
in your documentation is to use the *Install your project
inside a virtualenv using setup.py install* option in the admin panel of
your project
```

to link to the config file option.